.github/workflows/update_sdk_methods.py

```diff
@@ -0,0 +1,842 @@
+    "movement_sensor": {
+        "url": "https://raw.githubusercontent.com/viamrobotics/api/main/component/movementsensor/v1/movementsensor_grpc.pb.go",
+        "name": "MovementSensorServiceClient",
+        "methods": []
+    },
+    "power_sensor": {
+        "url": "https://raw.githubusercontent.com/viamrobotics/api/main/component/powersensor/v1/powersensor_grpc.pb.go",
+        "name": "PowerSensorServiceClient",
+        "methods": []
+    },
+    "sensor": {
+        "url": "https://raw.githubusercontent.com/viamrobotics/api/main/component/sensor/v1/sensor_grpc.pb.go",
+        "name": "SensorServiceClient",
+        "methods": []
+    },
+    "generic_service": {
+        "url": "https://raw.githubusercontent.com/viamrobotics/api/main/service/generic/v1/generic_grpc.pb.go",
+        "name": "GenericServiceClient",
+        "methods": []
+    },
+    "mlmodel": {
+        "url": "https://raw.githubusercontent.com/viamrobotics/api/main/service/mlmodel/v1/mlmodel_grpc.pb.go",
+        "name": "MLModelServiceClient",
+        "methods": []
+    },
+    "motion": {
+        "url": "https://raw.githubusercontent.com/viamrobotics/api/main/service/motion/v1/motion_grpc.pb.go",
+        "name": "MotionServiceClient",
+        "methods": []
+    },
+    "navigation": {
+        "url": "https://raw.githubusercontent.com/viamrobotics/api/main/service/navigation/v1/navigation_grpc.pb.go",
+        "name": "NavigationServiceClient",
+        "methods": []
+    },
+    "slam": {
+        "url": "https://raw.githubusercontent.com/viamrobotics/api/main/service/slam/v1/slam_grpc.pb.go",
+        "name": "SLAMServiceClient",
+        "methods": []
+    },
+    "vision": {
+        "url": "https://raw.githubusercontent.com/viamrobotics/api/main/service/vision/v1/vision_grpc.pb.go",
+        "name": "VisionServiceClient",
+        "methods": []
+    },
+    "app": {
+        "url": "https://raw.githubusercontent.com/viamrobotics/api/main/app/v1/app_grpc.pb.go",
+        "name": "AppServiceClient",
+        "methods": []
+    },
+    "data": {
+        "url": "https://raw.githubusercontent.com/viamrobotics/api/main/app/data/v1/data_grpc.pb.go",
+        "name": "DataServiceClient",
+        "methods": []
+    },
+    "robot": {
+        "url": "https://raw.githubusercontent.com/viamrobotics/api/main/robot/v1/robot_grpc.pb.go",
+        "name": "RobotServiceClient",
+        "methods": []
+    },
+    "mltraining": {
+        "url": "https://raw.githubusercontent.com/viamrobotics/api/main/app/mltraining/v1/ml_training_grpc.pb.go",
+        "name": "MLTrainingServiceClient",
+        "methods": []
+    }
+}
+
+## Language-specific resource name overrides:
+##   "proto_resource_name" : "language-specific_resource_name"
+##   "as-it-appears-in-type-array": "as-it-is-used-per-sdk"
+## Note: Always remap generic component and service, for all languages,
+##       as this must be unique for this script, but is non-unique across sdks.
+go_resource_overrides = {
```

Comment:
for all these language-specific configs, it would be a good idea to move these into a per-language file that gets imported at the top. That makes it easier to read and separate by language